src/components/Header/index.js

```diff
@@ -1,129 +1,154 @@
-    const show    = document.getElementsByClassName('sideBarLinkShow')[0];
-    const hide    = document.getElementsByClassName('sideBarLinkHide')[0];
+  render() {
+    const { openNotification } = this.state;
 
-    if (sideBar.style.display === 'none') {
-      sideBar.style.display = 'block';
-      show.style.display = 'none';
-      hide.style.display = 'block'
-    } else {
-      sideBar.style.display = 'none';
-      show.style.display = 'block';
-      hide.style.display = 'none'
-    }
+    return (
+      <Box fill>
+        <Box
+          as="header"
+          direction="row"
+          align="center"
+          pad={{ vertical: "small", horizontal: "medium" }}
+          justify="between"
+          background={{image: "url(https://seium.org/assets/backgrounds/light-stuff.png)"}}
+          elevation="large"
+          style={{ zIndex: "1000" }}
+        >
+          <Button href="/">
+            <Image width="100" height="50" src={require('./moonstone-logo.png')} />
+          </Button>
+          <Button
+            onClick={() => this.setState({ openNotification: !openNotification })}
+            icon={<Menu color='white'/>}
+          />
+        </Box>
+        <Box flex direction="row">
+          <Collapsible className="fullHeight" direction="vertical" open={openNotification}>
+            <Box
+              className="fullHeight"
+              width="xlarge"
+              background={{image: "url(https://seium.org/assets/backgrounds/light-stuff.png)"}}
+              pad="small"
+              elevation="small"
+            >
+              {this.props.content}
+            </Box>
+          </Collapsible>
+        </Box>
+      </Box>
+    );
   }
+}
 
-  render() {
-    const { openMenu } = this.state;
-    const loggedIn = localStorage.getItem("jwt") !== null;
+class Header extends Component {
+  renderEntries(paddingBottom) {
     return (
       <Box>
-          <Box gridArea="header" onClick={Header.hideAndShow} background="brand" className="sideBarLinkShow">
-            <Text><FaAngleDoubleRight /></Text>
+        {PAGES.map((page, i) => (
+          <Box key={i} pad={{horizontal: "large", bottom: paddingBottom}}>
+            <RoutedButton path={page.href}>
+              <Text color="white" size="large">{page.fa} {page.label}</Text>
+            </RoutedButton>
           </Box>
-          <Box gridArea="header" onClick={Header.hideAndShow} background="brand" className="sideBarLinkHide">
-            <Text><FaAngleDoubleLeft /></Text>
+        ))}
+      </Box>
+    )
+  }
+
+  render() {
+    const loggedIn = localStorage.getItem("jwt") !== null;
+
+    const content = (
+      <Box>
+          <Box align="center" pad={{ horizontal: "small", bottom: "medium" }}>
+            {this.props.size === "small" ? (
+              <Box pad={{top: "large"}} >
+                <QRCode
+                  renderAs="svg"
+                  value={"https://intra.seium.org/user/" + this.props.user.id}
```

Comment:
Should be https://moonstone.seium.org/user/
Maybe add the url to the config file?